src/sentry/plugins/sentry_mail/models.py

```diff
@@ -28,6 +28,9 @@ class MailPlugin(NotificationPlugin):
     project_conf_form = None
     mail_adapter = MailAdapter()
 
+    def rule_notify(self, event, futures):
+        return self.mail_adapter.rule_notify(event, futures)
+
```

Comment:
Where is this being used? The other PRs seemed to have the function already existing with its content replaced with a call to the mail adapter.

Reply:
Other than the test :-)

Reply:
Since `MailPlugin` is a `NotificationPlugin` we're just overriding the implementation of `NotificationPlugin.rule_notify` and calling `MailAdapter.rule_notify` instead. Actually realised I could simplify `NotificationPlugin.rule_notify` here too, might do that.